src/auth/index.ts

```diff
@@ -21,8 +21,7 @@ async function authenticateUser(
       });
     }
 
-    const em = (req as any).orm.em.fork();
-    const user = await em.findOne(User, { id: userId });
+    const user = await User.findOne({ _id: userId }).exec();
```

Comment:
when you only want to query documents by id , it is better to use the `findById()` method